scalabel/tests/test_scalabel_api.py

```diff
@@ -0,0 +1,73 @@
+""" Test file for the scalabel api """
```

Comment:
Test file usually in the format of "*_test.py".

Reply:
Renamed to match the convention